### ISSUE TITLE
Hauptmenü nimmt auf Mobilgeräten viel Platz ein

### ISSUE THREAD

**Issue by svenseeberg** [verdigado/sunflower#742]
## Describe the bug
In der Mobil-Ansicht wird für das Hamburger-Menü eine komplette weiße Zeile frei gehalten. Das nimmt, wenn man ganz oben auf der Seite ist, relativ viel Platz (fast 10%?) auf den nicht besonders großen Mobil-Geräten ein.

### Website or Screenshots

![image](https://github.com/user-attachments/assets/2a3e0607-9d56-4496-a23f-737d8a404d64)

### Environment
- Device: Desktop mit Mobilansicht
- OS: Linux
- Browser: Firefox
- Version of the sunflower theme: 2.2.0

## Additional context
N/A

## Possible Fix

Ich habe noch keinen voll ausgearbeiteten Vorschlag, wie es besser gemacht werden könnte. Möglicherweise lässt sich das Menü mit in den grünen Banner verschieben und dann mit dem Icon zusammen in einen Sticky Banner verlagern, sobald nach unten gescrollt wird.

**Comment by albig** [comment]
Ja, In der Ansicht gehört der Hamburger IMHO in den grünen Bereich. Das gibt aber Konflikte mit den meist längeren Titeln.

Sobald man scrollt, rutscht der Hamburger ja schon jetzt nach oben. 

Schnell umgebaut ist das nicht bzw. man muss dabei die Desktop-Ansicht berücksichtigen, wo das Menü ja unter dem Grünen Bereich bleiben soll.

Eventuell zusammen betrachten mit #728 ("gelber Balken" wie bei gruene.de) in einem Release 2.3.0.
